_posts/en/2020-05-20-Team-Topologies-Whitelisting-app.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: "Team Topologies Inspired Whitelist App Development"
+ref: tt-whitelist
+lang: en
+author: "IT Strategy Team"
+date: "2020-05-20"
+last_modified: "2020-05-20"
+excerpt_separator: <!--more-->
+---
+
+## Context
+
+During the COVID19 situation there was an urgent need for an application to assist with the management of important information regarding the internal use of resources within the department. An application was built to avoid compiling an unmanageable list of spreadsheets with inconsistent data. The IT Strategy Team alongside the DTS team worked to create a solution using modern approaches and methodologies, which are described below. By using the below model, the team was informally formed on a Thursday late afternoon and was able to begin demoing the application to users the following Monday -- four days later (having worked through the weekend). By Tuesday, five days after work began, the team had several repositories worth of work, and a deployment pipeline automatically deploying changes to a development environment, and, at the time, manually triggering deployments to staging and production. On the tenth day of work the team was able to demo the application to the CIO. Less than a month later, the PB Cloud environment became available at 11am, and within minutes we had the environment up and running, and were able to have users on the application to test by 1:00pm, only two hours later.
```

Comment:
Unnecessary comma usage and some run on sentences

>  The IT Strategy Team alongside the DTS team worked to create a solution using modern approaches and methodologies, which are described below

No need for comma here I think 

>  By Tuesday, five days after work began, the team had several repositories worth of work, and a deployment pipeline automatically deploying changes to a development environment, and, at the time, manually triggering deployments to staging and production.

By Tuesday, five days after the work began, the team had several repositories worth of work. An automated deployment pipeline was created in order to deploy changes to a development environment. Staging and production environments were also set up but deployment to these was still manually triggered.